pyaedt/misc/spisim_com_configuration_files/com_parameters.py

```diff
@@ -0,0 +1,375 @@
+        temp["io_control"] = self.io_control
+        temp["operational"] = self.operational
+        temp["tdr_and_erl_options"] = self.tdr_and_erl_options
+        temp["noise_jitter"] = self.noise_jitter
+        temp["table_93a3"] = self.table_93a3
+        temp["table_92_12"] = self.table_92_12
+        temp["floating_tap_control"] = self.floating_tap_control
+        temp["icn_fom_ild_parameters"] = self.icn_fom_ild_parameters
+        temp["receiver_testing"] = self.receiver_testing
+        temp["spisim_control"] = self.spisim_control
+        temp["other_parameters"] = self.other_parameters
+
+        with open(file_path, "w", encoding="utf-8") as f:
+            f.write(json.dumps(temp, indent=4, ensure_ascii=False))
+
+    @pyaedt_function_handler
+    def load(self, file_path):
+        """Load COM parameters from a json file.
+
+        Parameters
+        ----------
+        file_path : str,
+            Path of file.
+        """
+        with open(file_path) as f:  # pragma: no cover
+            temp = json.load(f)
+
+        for k, v in temp.items():  # pragma: no cover
+            for k2, v2 in v.items():
+                self.__getattribute__(k)[k2] = v2
+
+    @pyaedt_function_handler
+    def export_spisim_cfg(self, file_path):
+        """Export COM parameter to a SPISim cfg file.
+
+        Parameters
+        ----------
+        file_path : str
+            Path of file.
+        """
+        with open(file_path, "w") as fp:
+            fp.write("################################################################################\n")
+            fp.write("# MODULE: COM\n")
+            fp.write("# GENERATED ON\n")
+            fp.write("################################################################################\n")
+            for kw, v in self.parameters.items():
+                if kw in spimsim_matlab_keywords_mapping:
+                    kw = spimsim_matlab_keywords_mapping[kw]
+                fp.write("# {0}: {0}\n".format(kw.upper()))
+                fp.write("{} = {}\n".format(kw.upper(), v))
+        return True
+
+    @pyaedt_function_handler
+    def load_spisim_cfg(self, file_path):
+        """Load a SPIsim configuration file.
+
+        Parameters
+        ----------
+        file_path: str
+            Path of the configuration file.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        reverse_map = {j: i for i, j in spimsim_matlab_keywords_mapping.items()}
+
+        with open(file_path, "r") as fp:
+            lines = fp.readlines()
+            for line in lines:
+                if not line.startswith("#") and "=" in line:
+                    split_line = [i.strip() for i in line.split("=")]
+                    kw, value = split_line
+                    if kw in reverse_map:  # Get Matlab keyword
+                        kw = reverse_map[kw]
+                    self.set_parameter(kw, value)
+        return
```

Comment:
```suggestion
        return True
```
You should probably add a try except to return `False` if something goes wrong ?